src/models/addw1/README.md

```diff
@@ -17,12 +17,14 @@ The System76 Adder WS is a laptop with the following specifications:
     - MX25L12873F flash chip running System76 Firmware (non-open)
 - EC
     - ITE IT8587E running non-open firmware
-- GPU
-    - NVIDIA GeForce RTX 2070
-    - eDP 15.6-inch 3840x2160 OLED
-    - 1 x HDMI (w/HDCP) output
-    - 1 x Mini DisplayPort (1.3) output
-    - 1 x DisplayPort (1.3) over USB 3.1 Type C output
+- Graphics
+    - GPU: NVIDIA GeForce RTX 2070
+    - eDP display: 15.6" 3840x2160@60Hz OLED
+        - LCD panel: Samsung ATNA56WR06 (or equivalent)
```

Comment:
Should "LCD" be changed to "OLED"?

(also applies to addw2 and the OLED option on 15" oryp7)

Reply:
I guess that makes a lot of sense, done with 6f15fe8. (I'm now questioning whether we need LCD/OLED on both lines instead of just one, but leaving it out from either line doesn't seem right, so I'll leave it for now.)